### PR TITLE
GraphQL: RepoListWithNewestImage to return list of RepoSummary

### PR DESCRIPTION
**Pull request by chofnar**
**What type of PR is this?**
ui

**Which issue does this PR fix**:
#663

**What does this PR do / Why do we need it**:
From #663:

> ImageListWithLatestTag currently returns a list of ImageInfo objects. It needs to return consistent results with the API used for Global search as the same information will be used by the UI in the same type or cards.

Needed schema changes from #696 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by andaaron — CHANGES_REQUESTED** [review]
Actually can you update the implementation to match what Alex did for https://github.com/project-zot/zot/pull/695?
See my comments there about iterating over manifests and tags using the same index, which may or may not be the same in practice.

**Review by andaaron — DISMISSED** [review]

**Review by andaaron — COMMENTED** [review]
In case there are errors, let's have the query return a partial list of "good" repos, and a list of all the errors encountered.
We don't want to break the entire query results for a few bad repos/images.

File: pkg/extensions/search/common/common_test.go [hunk 1]
```diff
@@ -55,9 +56,9 @@ var (
 	subRootDir string
 )
 
-type ImgResponsWithLatestTag struct {
-	ImgListWithLatestTag ImgListWithLatestTag `json:"data"`
-	Errors               []ErrorGQL           `json:"errors"`
+type RepoResponseWithNewestImage struct {
```
Comment (rchincha) [inline pkg/extensions/search/common/common_test.go:59]: Call this `RepoWithNewestImageResponse` instead

Comment (chofnar) [reply pkg/extensions/search/common/common_test.go]: Renamed

**Review by andaaron — APPROVED** [review]

**Comment by rchincha** [comment]
Pls fix the merge conflicts and resubmit.

**Review by andaaron — APPROVED** [review]

**Pull request merged**